src/components/application_manager/src/helpers/application_helper.cc

```diff
@@ -19,9 +19,9 @@ void DeleteWayPoints(ApplicationSharedPtr app,
 
 void DeleteCommands(ApplicationSharedPtr app, ApplicationManager& app_manager) {
   auto accessor = app->commands_map();
-  const auto& commands_map = accessor.GetData();
+  const auto commands_map = accessor.GetData();
 
-  for (const auto& cmd : commands_map) {
+  for (const auto cmd : commands_map) {
```

Comment:
```suggestion
  for (const auto& cmd : commands_map) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e